app/pages/lab-pages-editor/components/WorkflowSettingsPage.jsx

```diff
@@ -47,7 +47,12 @@ export default function WorkflowSettingsPage() {
   if (!workflow) return null;
 
   return (
-    <form className="workflow-settings-page" onSubmit={onSubmit}>
+    <form
+      aria-labelledby="pages-editor_workflow-header-tab-button_settings"
+      className="workflow-settings-page"
+      onSubmit={onSubmit}
+      role="tabpanel"
```

Comment:
This breaks the form by overriding its default form role.

Reply:
Oh dang, that's good point - I'll fix this right away.